### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by sorja**
part of #1092 
image

Todo
- [x] Navigation state (`store.navigation` vs `store.ui.navigation`)
- [x] NavigationHeader
- [ ] ReviewStatusMarker
- [ ] Header/Status
- [x] Expand on url match

### CODE REVIEW

File: client/components/Navigation/Navigation.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
```
Comment (minotogna) [inline client/components/Navigation/Navigation.tsx:2]: why not to add all old code and comment it out?

**Review by minotogna — CHANGES_REQUESTED** [review]

File: client/components/Navigation/Navigation.tsx (hunk 1 above)
Comment (sorja) [reply client/components/Navigation/Navigation.tsx:2]: Added files for what I need, but can add all the code too

Comment (minotogna) [reply client/components/Navigation/Navigation.tsx:2]: yes please, so we know what we'll need to do

**Review by minotogna — CHANGES_REQUESTED** [review]
add Navigation state + expand on url match + navigation header (comment things will implement later)

File: client/store/rootReducer.ts [hunk 2]
```diff
@@ -1,9 +1,11 @@
 import AssessmentSlice from './assessment/slice'
 import LoginSlice from './login/slice'
 import UserSlice from './user/slice'
+import UiSlice from './ui/slice'
 
 export default {
   assessment: AssessmentSlice,
   login: LoginSlice,
   user: UserSlice,
+  ui: UiSlice,
```
Comment (minotogna) [inline client/store/rootReducer.ts:10]: store should be ui/navigation.

export actions/hooks/reducer etc from store/ui/navigation 
e.g. ```import {NavigationActions} from '@store/ui/navigation'```

**Review by minotogna — CHANGES_REQUESTED** [review]

**Review by minotogna — APPROVED** [review]

**Pull request merged**